packages/block-library/src/navigation/edit/edit-overlay-button.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+import { store as coreStore, useEntityProp } from '@wordpress/core-data';
+import { parse, serialize } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { useHistory } = unlock( routerPrivateApis );
+
+export default function EditOverlayButton( { navRef } ) {
+	// TODO: get the slug of the currently active theme
+	const themeSlug = 'emptytheme';
```

Comment:
Maybe like [this](https://github.com/WordPress/gutenberg/blob/1e3dd2a620a71499abb79484a939d10376ff25d0/packages/edit-site/src/components/sidebar-navigation-screen-template/index.js#L31)?

Reply:
✅